test/primitives.jl

```diff
@@ -58,13 +58,16 @@
     @test normal(p) == Vec(0, 0, 1)
     @test isnothing(boundary(p))
 
-    p = Plane(P3(0, 0, 0), V3(0, 0, 1))
-    @test p(T(1), T(0)) == P3(1, 0, 0)
-    @test p(T(0), T(1)) == P3(0, 1, 0)
```

Comment:
Can you please elaborate on the new results? The previous implementation wasn't correct? These tests that were erased seemed correct in behavior.